content/webapp/pages/article-series.tsx

```diff
@@ -49,25 +69,11 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
 
     const articlesQuery = await fetchArticles(client, {
       predicates: [prismic.predicate.at(seriesField, id)],
-      page: 1,
-      pageSize: 100,
+      page,
+      pageSize: 20,
```

Comment:
This is the same as the number of articles we fetch on a single page; 100 entries is _loooooooong_.

In practice the majority of series are a six-part serial and so this doesn't make a difference either way.